composer.json

```diff
@@ -72,6 +72,14 @@
             ],
             "locations": {
                 "web-root": "web/"
+            },
+            "file-mapping": {
+            "[web-root]/sites/development.services.yml": {
```

Comment:
The indention is wrong.

Reply:
Thanks for checking this out, fixed the indentation